browser/resources/brave_new_tab_page_refresh/state/rewards_handler.ts

```diff
@@ -55,8 +54,7 @@ export function createRewardsHandler(
   async function updateBalance() {
     const { balance } = await rewardsHandler.getAvailableBalance()
     store.update({
-      rewardsBalance:
-          new Optional(typeof balance === 'number' ? balance : undefined)
+      rewardsBalance: typeof balance === 'number' ? balance : null
```

Comment:
isn't the type of `balance` here `number | null`? If so:
```suggestion
      rewardsBalance: balance
```

Reply:
Indeed - thanks!